src/helpers/api/index.ts

```diff
@@ -0,0 +1,42 @@
+import { PostTexts } from '@/models/Post'
+import CID from '@/models/CID'
+import env from '@/helpers/env'
+import fetchWithTimeout from '@/helpers/fetchWithTimeout'
+import structToCid from '@/helpers/structToCid'
+import validatePost from '@/helpers/api/validatePost'
+
+const cidToData = new Map<CID, unknown>([])
```

Comment:
Why do you need this, if you don't `export/return cidToData` and don't use it :thinking: